LICENSE.txt

```diff
@@ -1,199 +1,21 @@
+Copyright 2017 Harvard Medical School
```

Comment:
@JoostJM @hugoaerts @pieper thinking about this for dcmqi, I decided to list BWH as the copyright holder, see https://github.com/QIICR/dcmqi/blob/master/LICENSE.txt. The reason is the, in case of dcmqi, BWH is the recepient of the award from NIH. It might make sense to list DFCI here, since in the case of pyradiomics, DFCI is the recipient of the main award funding pyradiomics development.